spec/controllers/books_controller_spec.rb

```diff
@@ -0,0 +1,28 @@
+# frozen_string_literal: true
+
+require "rails_helper"
+
+RSpec.describe BooksController, type: :controller do
+
+  let(:book) { FactoryBot.create(:book) }
+
+  describe "GET #index" do
+    it "returns json when requested" do
```

Comment:
This says it expects json, but is returning html?

Reply:
typos from copying code.